### ISSUE TITLE
Error in 4.2.5.1 standard metadata names

### ISSUE THREAD

**Issue by LJWatson** [w3c/html#26]
Originally posted to public-html@w3.org [1].

In description of application-name,
Current text is :

```
<p>
The value must be a short free-form string giving the name of the Web application that the page represents.
If the page is not a Web application, the <code data-anolis-xref="meta-application-name"><a href="#meta-application-name">application-name</a></code> metadata name must not be used. 
Translations of the Web application's name may be given, using the <code data-anolis-xref="attr-lang"><a href="dom.html#attr-lang"></a></code> attribute to specify the language of each name.
</p>
```

i think that it shoud be :

```
<p>
The value must be a short free-form string giving the name of the Web application that the page represents.
If the page is not a Web application, the <code data-anolis-xref="meta-application-name"><a href="#meta-application-name">application-name</a></code> metadata name must not be used. 
Translations of the Web application's name may be given, using the <code data-anolis-xref="attr-lang"><a href="dom.html#attr-lang">lang</a></code> attribute to specify the language of each name.
</p>
```

[1] https://lists.w3.org/Archives/Public/public-html/2015Nov/0001.html

**Comment by ZoeBijl** [comment]
This seems to already be fixed in 5.1: [4.2.5.1. Standard metadata names in HTML 5.1](http://w3c.github.io/html/#standard-metadata-names).

Code from `semantics.include` (line 887 at the moment of writing):

```
: <dfn><code>application-name</code></dfn>
:: The value must be a short free-form string giving the name of the Web application that the page
    represents. If the page is not a Web application, the <code>application-name</code> metadata
    name must not be used. Translations of the Web application's name may be given, using the
    <{global/lang}> attribute to specify the language of each name.
```

**Comment by chaals** [comment]
Yes, seems we can close this. Thanks
